spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/filter/factory/SpringCloudCircuitBreakerFilterFactory.java

```diff
@@ -137,7 +146,7 @@ public String name() {
 
 		private String name;
 
-		private URI fallbackUri;
+		private String fallbackUri;
```

Comment:
This is a breaking change, not sure we can make this without being backwards compatible.

Reply:
I thought it was not a breaking change, because [here](https://github.com/spring-cloud/spring-cloud-gateway/blob/master/spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/filter/factory/SpringCloudCircuitBreakerFilterFactory.java#L162) the value came as string, I just convert to URI after apply possibles path variables.

Is there something that a can do to improve compatibility ?

Reply:
It's a public api, so a breaking change.

Reply:
Ok. I don't know how to do it enhancement in fallback without this change, because I cannot use placeholders in a URI.

I did a solution in my project using a custom filter like this: https://github.com/brunogs/fallback/blob/master/src/main/resources/application.yml#L5-L30

I thought it would be cool do the same with less code. 
But I'm not sure if this PR is the best solution, what do you think ? Is a good enhancement for the next versions ? 

If it doesn't seem to be a necessary improvement it is ok for me to close this PR.